docs/azureai/ai-integrations-compatibility-matrix.md

```diff
@@ -0,0 +1,196 @@
+---
+
+# Aspire AI integrations compatibility matrix
+
+.NET Aspire provides several AI hosting and client integrations that enable you to work with different AI services and platforms. This article provides a compatibility matrix showing which client integrations work with which hosting integrations, along with guidance on the recommended pairings.
+
+## Compatibility matrix
+
+The following table shows the compatibility between Aspire AI hosting and client integrations:
+
+| **Hosting Integration**                    | **Aspire.OpenAI**            | **Aspire.Azure.AI.OpenAI**                    | **Aspire.Azure.AI.Inference** |
+|--------------------------------------------|------------------------------|-----------------------------------------------|-------------------------------|
+| `Aspire.Hosting.Azure.AI.Foundry`          | ❌ No                       | ⚠️ Partial                                    | ✅ Yes (preferred)           |
+| `Aspire.Hosting.Azure.CognitiveServices`   | ❌ No                       | ✅ Yes (preferred)                            | ❌ No                        |
+| `Aspire.Hosting.OpenAI`                    | ✅ Yes (preferred)          | ✅ Yes                                        | ❌ No                        |
+| `Aspire.Hosting.GitHub.Models`              | ⚠️ Partial                  | ❌ No                                         | ✅ Yes (preferred)           |
+
+### Legend
+
+- ✅ **Yes**: The client integration is compatible with the hosting integration
+- ⚠️ **Partial**: The client integration might not support all models or APIs
+- ❌ **No**: The client integration does not support the hosting environment
+- **(preferred)**: The recommended client integration for this hosting integration
+
+## Recommended pairings
+
+In general, use **Aspire.Azure.AI.Inference** to connect to Azure hosted models, or **Aspire.OpenAI** to connect directly to OpenAI.
+
+### Azure AI Foundry
+
+For Azure AI Foundry resources, use the **Aspire.Azure.AI.Inference** client integration. This provides the best compatibility with the diverse range of models available through Azure AI Foundry.
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var foundry = builder.AddAzureAIFoundry("foundry");
+
+builder.AddProject<Projects.ExampleProject>()
+       .WithReference(foundry);
+```
+
+In your service, use the Azure AI Inference client:
+
+```csharp
+builder.AddAzureChatCompletionsClient("foundry");
+```
+
+### Azure Cognitive Services (Azure OpenAI)
+
+For Azure OpenAI resources, use the **Aspire.Azure.AI.OpenAI** client integration for full Azure-specific features and authentication support.
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var openai = builder.AddAzureOpenAI("openai");
+
+builder.AddProject<Projects.ExampleProject>()
+       .WithReference(openai);
+```
+
+In your service, use the Azure OpenAI client:
+
+```csharp
+builder.AddAzureOpenAIClient("openai");
+```
```

Comment:
We need to be clearer about which code belongs where. The opening sentence talks about client integration but the code that immediately follows if for the hosting integration. We're not showing the hosting integration package, each section should be more explicit as to avoid confusion.